casper/src/main/scala/coop/rchain/casper/util/comm/CasperPacketHandler.scala

```diff
@@ -353,6 +353,7 @@ object CasperPacketHandler extends CasperPacketHandlerInstances {
         isOldBlock <- MultiParentCasper[F].contains(b)
         _ <- if (isOldBlock) {
               Log[F].info(s"Received block ${PrettyPrinter.buildString(b.blockHash)} again.")
+              MultiParentCasper[F].refetchDependencies(b)
```

Comment:
Missing `*>` from previous effect (i.e. logging will not actually happen when `F` is not `Id`).